client/js/app/components/explorer/visualization/index.js

```diff
@@ -88,6 +89,12 @@ var Visualization = React.createClass({
     AppDispatcher.unregister(this.dispatcherToken);
   },
 
+  exportToCsv: function() {
```

Comment:
I think this has been resolved in my latest commit.. or maybe I'm not seeing the same thing.